src/libraries/Microsoft.PowerFx.Interpreter/Environment/PowerFxConfigExtensions.cs

```diff
@@ -20,5 +20,15 @@ public static void AddFunction(this PowerFxConfig powerFxConfig, ReflectionFunct
         {
             powerFxConfig.AddFunction(function.GetTexlFunction());
         }
+
+        public static void AddDefaultEnums(this PowerFxConfig powerFxConfig)
+        {
+            powerFxConfig.EnumStoreBuilder.WithDefaultEnums();
+        }
+
+        public static void AddRequiredEnums(this PowerFxConfig powerFxConfig)
+        {
+            powerFxConfig.EnumStoreBuilder.WithRequiredEnums(powerFxConfig.Functions);
+        }
```

Comment:
This is in Interpreter.
Can these be moved to Core?

Reply:
Moved